lib/store.js

```diff
@@ -436,3 +438,64 @@ Store.prototype.compose = function(
     callback(err, identity);
   });
 };
+
+/**
+ * Marks a Credential as deleted.
+ *
+ * @param actor the Identity performing the action.
+ * @param credentialId the Credential ID.
+ * @param status the status.
+ * @param callback(err) called once the operation completes.
+ */
+Store.prototype.remove = function(actor, credentialId, callback) {
+  var self = this;
+  async.auto({
+    find: function(callback) {
```

Comment:
Do we typically run `find` in a remove call? I suppose you'd get a permission denied error instead if we didn't...